apple/swift-docc-render
# Remove nav background flashing, when changing between color modes

Bug/issue #, if applicable: 90712731

## Summary

Extract the nav bg transition timing to a scss variable. This way clients can control the transition timing in their templates.

## Dependencies

NA

## Testing

Steps:
1. Go to a documentation page
2. Switch between color modes (light/dark)
3. Assert the nav no longer has a transition/delay in the background color
4. Assert that colors are still shifted where expected, like on TutorialOverview, where we apply a blur to the background

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
